test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/Dnu.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Dnu
+    {
+        private readonly string _sdkPath;
+
+        public Dnu(string sdkPath)
+        {
+            _sdkPath = sdkPath;
+        }
+
+        public ExecResult Publish(
+            string projectPath,
+            string outputPath,
+            bool noSource = false,
```

Comment:
It's not, especially since noSource is defaulted to false. I can imagine requiring this default argument for options we want to default true but this is not the case here.